src/nyc_trees/sass/partials/_survey.scss

```diff
@@ -1,12 +1,138 @@
+.page-new-survery {
+    .map-action-sidebar {
+        position: static;
+    }
+
+    .map-sidebar {
+        max-height: 75%;
+        overflow: scroll;
+        box-shadow: 0 1px 2px rgba(black, .2);
+    }
+}
+
+.map-survey {
+    position: fixed;
+    top: 5.6rem;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    @media (max-width: $screen-sm) {
+        bottom: 15rem; 
+    }
+}
+
+.action-bar-survey {
+    overflow-x: hidden;
+    .btn-group {
+        margin-bottom: .75rem;
+    }
+
+    @media (max-width: $screen-sm) {
+        position: fixed;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: 15rem;
+        padding: $grid-gutter-width/3 $grid-gutter-width;
+        z-index: 1;
+        background-color: #fff;
+        display: table;
+        .tab-status-list & {
+          display: none;
+        }
+        #select-starting-point, #select-side, #survey {
+            display: table-cell;
+            vertical-align: middle;
+        }
+
+        &.expanded {
+            top: 5.6rem;
+            overflow: scroll;
+            bottom: 0;
+            display: block;
+            height: auto;
+            padding: 0;
+        }
+
+    }
+}
+
+#tree-form-header {
+    #no-trees {
+        position: absolute;
+        top: 1.2rem;
+        right: 2rem;
+    }
+
+    @media (max-width: $screen-sm) {
+        padding: $grid-gutter-width/3 $grid-gutter-width;
+        background: $gray-lightest;
+        margin-bottom: 1rem;
+    }
+
+    @media (min-width: $screen-sm) {
+        h4 {
+            margin-top: 0;
+        }
+    }
+}
+
+#tree-form-container, .tree-form-block {
+    h3 {
+        margin-bottom: 2rem;
+    }
+
+    @media (max-width: $screen-sm) {
+        padding: $grid-gutter-width/3 $grid-gutter-width;
+    }
+}
+
 .tree-form {
+    label {
+        color: $gray;
+    }
     // Hide checkboxes and radios, and use their checked/unchecked to style their label
     input[type="checkbox"], input[type="radio"] {
         opacity: 0;
+        display: none;
```

Comment:
The JS for the treecorder only submits "visible" elements (anything that returns true for jQuery's `.is(":visible")`), which is why I gave this `opacity: 0`.  Setting this to `display: none` breaks actually submitting the form.

Unfortunately I can't think of a way to make jQuery think this element is "visible" without it messing up the layout.
Mind if I push a commit to this branch that will fix the JS, so we can keep the `display: none`?